CMakeLists.txt

```diff
@@ -79,16 +79,18 @@ if (NOT USE_UNOFFICIAL_OGRE_VERSIONS)
       COMPONENTS ${ign_ogre_components}
       REQUIRED_BY ogre
       PRIVATE_FOR ogre)
-  else()
-    # If ogre 1.10 or greater was not found, then proceed to look for 1.9.x
-    # versions which are offically supported
-    ign_find_package(IgnOGRE VERSION 1.9.0
-      COMPONENTS ${ign_ogre_components}
-      REQUIRED_BY ogre
-      PRIVATE_FOR ogre)
   endif()
 endif()
 
+if (NOT OGRE_FOUND)
+  # If ogre 1.10 or greater was not found, then proceed to look for 1.9.x
```

Comment:
This comment may be a bit misleading, as `OGRE_FOUND` is false either if ogre 1.10 or greater was not found, or also if `USE_UNOFFICIAL_OGRE_VERSIONS` is explicitly set to `TRUE` .